marketplace.Makefile

```diff
@@ -81,36 +81,26 @@ include $(makefile_dir)/var.Makefile
 	mkdir -p "$@"
 
 
-# Note: Run after bumping.
-MARKETPLACE_K8S_APP_TOOLS_HELM_HOOKS_VERSION := 0.0.2
-.build/marketplace/deployer/marketplace-k8s-app-tools-helm-hooks/marketplace-k8s-app-tools-helm-hooks-$(MARKETPLACE_K8S_APP_TOOLS_HELM_HOOKS_VERSION).tgz: \
-		$(MARKETPLACE_TOOLS_PATH)/marketplace/deployer_helm_tiller_base/marketplace-k8s-app-tools-helm-hooks/* \
-		.build/marketplace/delete_deprecated \
-		.build/marketplace/deployer/marketplace-k8s-app-tools-helm-hooks \
-		.build/var/MARKETPLACE_TOOLS_TAG \
-		| .build/marketplace/deployer
-	$(call print_target)
-	cd $(MARKETPLACE_TOOLS_PATH) \
-	&& helm package marketplace/deployer_helm_tiller_base/marketplace-k8s-app-tools-helm-hooks/ \
-	      --version "$(MARKETPLACE_K8S_APP_TOOLS_HELM_HOOKS_VERSION)" \
-	      --destination .build/marketplace/deployer/marketplace-k8s-app-tools-helm-hooks \
-	&& helm gcs push .build/marketplace/deployer/marketplace-k8s-app-tools-helm-hooks/marketplace-k8s-app-tools-helm-hooks-$(MARKETPLACE_K8S_APP_TOOLS_HELM_HOOKS_VERSION).tgz cloud-marketplace-tools --force
+.build/marketplace/charts: | .build/marketplace
+	mkdir -p "$@"
 
-.build/marketplace/deployer/helm_tiller/tester: .build/marketplace/deployer/helm_tiller
+CHART_BUCKET = marketplace-k8s-app-tools-charts
+MARKETPLACE_HOOKS_VERSION = 0.0.1
+.build/marketplace/charts/marketplace-hooks: \
+		.build/var/CHART_BUCKET \
+		.build/var/MARKETPLACE_HOOKS_VERSION \
+		marketplace/charts/Dockerfile \
+		$(shell find marketplace/charts/ -type f) \
+		$(MARKETPLACE_TOOLS_PATH)/marketplace/charts/marketplace-hooks/* \
+		| .build/marketplace/charts
 	$(call print_target)
 	cd $(MARKETPLACE_TOOLS_PATH) \
 	&& docker build \
```

Comment:
so we're building a docker image to get a side effect of publishing the chart?

Reply:
Yes - in this way we can standardize the environment in which we build/publish charts.

Reply:
Let's not add release scripts to this makefile. This is intended for local development only. Note that non of our make targets currently update production artifacts, and it's a good thing. Production artifacts are updated by cloudbuild. Let's move this there.

Reply:
I'll stage chart release process changes separately. :)